src/components/shared/Promotions/Promotions.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import { injectIntl } from 'react-intl';
+import { InjectAppServices } from '../../../services/pure-di';
+import Loading from '../../Loading/Loading';
+
+/**
+ * Promotions
+ * @param { Object } props
+ * @param { import('react-intl').InjectedIntl } props.intl
+ * @param { import('../../services/pure-di').AppServices } props.dependencies
+ */
+const Promotions = function({ intl, type, page, dependencies: { dopplerLegacyClient } }) {
+  const [bannerData, setBannerData] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
Don't you need to handle possible errors also here? Maybe I'm loosing something.

Reply:
for example??

Reply:
Connection error, timeout, service down, etc.

In my opinion, the DopplerSitesClient should catch all exceptions and return a default data.

Reply:
maybe need to check with sites people

Reply:
Si, con Martín, pero en cualquier caso, creo que tenemos que manerjar el error de una forma elegante.